### PR TITLE
Ohm

### PR DESCRIPTION
**Pull request by watcharapol28**

### CODE REVIEW

File: src/components/commons/Badge/index.tsx [hunk 1]
```diff
@@ -8,4 +8,5 @@ const Badge = styled.div`
   background-color: var(--pamary-color);
 `
 
+
```
Comment (Acup0fTea) [inline src/components/commons/Badge/index.tsx:11]: delate

File: src/containers/bill/constants.ts [hunk 2]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
 export const dataProducts: Product[] = [
   {
     name: 'โค้ก 1.25L'
+    price: 340
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:6]: comma

File: src/containers/bill/constants.ts [hunk 3]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
...
+    quantity: 1
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:7]: comma

File: src/containers/bill/constants.ts [hunk 4]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
...
   },
   {
     name: 'เลย์ 1 แพ็ค'
+    price: 300
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:11]: comma

File: src/containers/bill/constants.ts [hunk 5]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
...
+    quantity: 1
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:12]: comma

File: src/containers/bill/constants.ts [hunk 6]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
...
   },
   {
     name: 'โก้แก่ 1 ลัง'
+    price: 500
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:16]: comma

File: src/containers/bill/constants.ts [hunk 7]
```diff
@@ -3,11 +3,17 @@ import { Product } from '@containers/bill/interface'
...
+    quantity: 1
```
Comment (Acup0fTea) [inline src/containers/bill/constants.ts:17]: comma

**Comment by watcharapol28** [comment]
finish

**Pull request closed without merging**